docs/neighbor-view-mutation-survivors.md

```
@@ -0,0 +1,41 @@
+python -m mutmut results
+```
+
+## Result
+
+| Run | Total | Killed | Survived |
+| --- | ---: | ---: | ---: |
+| Baseline `jitter_coordinates` slice | 59 | 48 | 11 |
+| After #498 | 59 | 54 | 5 |
+
+The 11 baseline survivors were `jitter_coordinates` mutants #5, #6, #8, #9,
+#10, #15, #23, #38, #49, #50, #51.
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)